monitoring/oplog-populator/dashboard.py

```diff
@@ -0,0 +1,335 @@
+from grafanalib.core import (
+    ConstantInput,
+    DataSourceInput,
+    RowPanel,
+    Threshold,
+    Templating,
+    Template,
+)
+from grafanalib import formatunits as UNITS
+from scalgrafanalib import(
+    metrics,
+    layout,Target,
+    TimeSeries,
+    Dashboard,
+    Stat,
+    PieChart,
+)
+
+class Metrics:
+    RECONFIG_LAG = metrics.BucketMetric(
+        'oplog_populator_reconfiguration_lag_sec',
+        'job', 'connector', 'namespace'
+    ).with_defaults(
+        'job="${oplog_populator_job}"',
```

Comment:
this can be added as named parameter in the constructor parameters:

```
etrics.BucketMetric(
        'oplog_populator_reconfiguration_lag_sec',
        'connector', 'namespace', job="${oplog_populator_job}"
    ).with_defaults('connector=~"${connector}"', 'namespace="${namespace}"')